### ISSUE TITLE
Map & data sharing roadmap

### ISSUE THREAD

**Issue by RohanClarke** [Regen-Digital/farm_regen_digital#8]
1. Define key maps for including in module
- https://github.com/paul121/farm_regen_digital/issues/9
- Meeting with SQL 24Aug22 - identify key mapping requirements (maps, sources, type, API)

2. Scope features (UI/UX) - for using the maps... 
- https://github.com/paul121/farm_regen_digital/issues/10
- https://github.com/paul121/farm_regen_digital/issues/11
- https://github.com/paul121/farm_regen_digital/issues/12

3. https://github.com/paul121/farm_regen_digital/issues/13
- Authorising (and revoking) the sharing of data with specific instances
- Identifying which data is to be shared and any constraints
- Displaying the aggregated data in the parent (recipient) instance

**Comment by paul121** [comment]
@RohanClarke I've created separate issues for some of the bullet points above. Happy to continue using this as a general tracking issue as we continue to have "roadmap" level ideas. Lets continue revisiting & breaking out to concrete issues as needed

**Comment by paul121** [comment]
> Wardy says:Use cases for aggregation

Non-farmers within singular project - require them all to access, contribute and map (Might work as current twin with many people added)
Farmer with Multiple farms - example requires columns for logs by farm 1, Farm 2 and Farm x
Advisor with lots of farmers - Many - possibly too many twins to view within the twin.
Processor with lots of suppliers 
Wardy says:I want to share the individual results from the same survey to an aggregation.  I want to take the log entries of a certain type and share them.  I want to share polygon (boundary) for multiple twins with 1 twin.
